examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -223,4 +227,11 @@ def plot_matrices(matrices, matrix_kind):
 plt.grid(True)
 plt.tight_layout()
 
+###############################################################################
+# While the comparison is not fully conclusive on this small dataset,
```

Comment:
Should we suggest here that users can re-run the example with all available data and compare ?